src/main/docs/guide/server.adoc

```diff
@@ -154,7 +153,7 @@ The above example uses the `@GrpcChannel` annotation to inject a gRPC `ManagedCh
 
 Now that you have a test client, writing the test becomes trivial:
 
-snippet::helloworld.GreetingEndpointTest[tags="imports,test", project-base="examples/hello-world"]
+snippet::helloworld.GreetingEndpointTest[tags="imports,test", project-base="test-suite"]
```

Comment:
if you use `test-suite`, you don't need to specify a `project-base` for the `snippet` macro.

Reply:
Done! Also updated the version to 3.2.0-SNAPSHOT

Reply:
Done 👍